massdriver/resource_package_alarm.go

```diff
@@ -8,9 +8,17 @@ import (
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
 )
 
+type PackageAlarmMetric struct {
+	Name       string            `json:"name"`
+	Namespace  string            `json:"namespace,omitempty"`
```

Comment:
leaving namespace and dimensions as `omitempty` meaning they will be missing from the payload if they aren't populated.

Reply:
Both `namespace` and `name` should be required.

`statistic` is a concept in AWS and Azure but not in GCP, so it shouldn't be required right now.